webapp-django/crashstats/topcrashers/jinja2/topcrashers/topcrashers.html

```diff
@@ -16,8 +16,8 @@
                 <a href="{{ url(
                     'crashstats:topcrasher',
                     product=query['product'],
-                    versions=query['versions'],
-                    date_range_type='report',
+                    versions=';'.join(query['versions']),
```

Comment:
I think that's basically what we have been doing up to now, based on the same arguments that you wrote (not a priority, technical audience... ). :)